### PR TITLE
Feature/expose primary file version [PREP-144]

### PR DESCRIPTION
**Pull request by pattisdr**
## Ticket

Needed for https://openscience.atlassian.net/browse/PREP-144

## Purpose

The version of the primary file for the preprint needs to be exposed in the view preprint page.

## Changes 

Adds version of the selected file and a download link for the selected file to the view preprint page.  Download preprint button just downloads primary file.

screen shot 2016-09-27 at 11 42 29 am

### CODE REVIEW

**Comment by caneruguz** [comment]
@pattisdr thanks for the work. We probably need to talk to product team about functionality and location. I'll coordinate with them.

**Comment by pattisdr** [comment]
@caneruguz there are two other PR's needed alongside this, one for Ember-OSF and one for OSF. Links in ticket.

**Comment by caneruguz** [comment]
@pattisdr I moved it to demo lane in JIRA. Let's check in with product first before code review

File: app/templates/components/supplementary-file-browser.hbs [hunk 1]
```diff
@@ -2,8 +2,19 @@
     width="99%" height="700"}}
 {{/file-renderer}}
 
+<div class="row">
+    <div class="col-xs-8">
+        <p class="f-w-lg p-b-md">{{selectedFile.name}}</p>
+    </div>
+    <div class="col-xs-4 pull-right text-nowrap">
```
Comment (caneruguz) [inline app/templates/components/supplementary-file-browser.hbs]: don't pull-right the col-xs-4, make a div inside it and pull that to the right or right align the text.

**Pull request merged**